server/routerlicious/packages/services/src/redis.ts

```diff
@@ -26,6 +28,7 @@ export class RedisCache implements ICache {
 
         client.on("error", (err) => {
             winston.error("Error with Redis:", err);
+            Lumberjack.error("Error with Redis:", err);
```

Comment:
`Lumberjack.error`'s second positional parameter is a properties parameter. Error should be the third parameter, so the above might not produce the desired effect. Can you do instead:
```typescript
Lumberjack.error("Error with Redis:", undefined, err);
```
https://github.com/microsoft/FluidFramework/blob/4202d915b7f92f9afeb607219cb5369b362905a1/server/routerlicious/packages/services-telemetry/src/lumberjack.ts#L90